src/main/scala/beam/agentsim/agents/ridehail/AlonsoMoraPoolingAlgForRideHail.scala

```diff
@@ -177,8 +177,8 @@ object AlonsoMoraPoolingAlgForRideHail {
     solutionSpaceSizePerVehicle: Int
   ): List[(RideHailTrip, VehicleAndSchedule, Double)] = {
     import scala.collection.mutable.{ListBuffer => MListBuffer}
-    val Rok = MListBuffer.empty[CustomerRequest]
-    val Vok = MListBuffer.empty[VehicleAndSchedule]
+    val Rok = collection.mutable.HashSet.empty[CustomerRequest]
```

Comment:
@haitamlaarabi please, take a look to these changes. It should not break anything, but still.